tasks/src/main/java/org/example/aStartUp2/AStartUp2.java

```diff
@@ -0,0 +1,41 @@
+package org.example.aStartUp2;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+/**
+ * https://codeforces.com/problemset/problem/420/A
+ * Solution with Collections (Set), loops and Scanner
+ * Question:
+ * What is faster to read? A Set (HashSet) with complexity probably O(1) or a List (ArrayList) with complexity probably O(n)?
+ */
+public class AStartUp2 {
+    Scanner sc = new Scanner(System.in);
+    String nameOfFirm = sc.nextLine();
+    //sc.close();
+
+    public boolean aStartUp2(String nameOfFirm) {
+        Set<Character> symmetricalLetters = new HashSet<>();
+        symmetricalLetters.add('A');
```

Comment:
lepiej uzyc Set.of(A,B,C) - jest to niemutowalna wersja seta ale Ty tylko z niego czytasz wiec nie ma problemu i zaoszczedza 20 linii kodu